packages/client-node/src/index.ts

```diff
@@ -46,4 +46,17 @@ export {
   StreamableJSONFormats,
   SingleDocumentJSONFormats,
   RecordsJSONFormats,
+  type SimpleColumnType,
+  type ParsedColumnSimple,
+  type ParsedColumnEnum,
+  type ParsedColumnFixedString,
+  type ParsedColumnNullable,
+  type ParsedColumnDecimal,
+  type ParsedColumnDateTime,
+  type ParsedColumnDateTime64,
+  type ParsedColumnArray,
+  type ParsedColumnTuple,
+  type ParsedColumnMap,
+  type ParsedColumnType,
+  parseColumnType,
```

Comment:
Neither nodejs nor web clients use any of these by default. Should we consider releasing it as a separate package? 
What benefits does the current distribution model have? I see a few disadvantages: larger distro size and harder to explore the original source code

Reply:
> Neither nodejs nor web clients use any of these by default. Should we consider releasing it as a separate package?

If we are ever to add Native or RowBinaryWithNamesAndTypes, then this parser will be required anyway. Or do you mean that it should not be exported from the Node.js client, only in common?

> harder to explore the original source code

WDYM?